### PR TITLE
ram instruction edits

### PR DESCRIPTION
**Pull request by jvaidy3**

### CODE REVIEW

File: docs/content/docs/how-to-guides/1_deployEnbuildQuickstart.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ seo:
 Follow these step-by-step instructions to deploy ENBUILD locally for testing.
 
 ## Prerequisites
+Before you begin, ensure that you have the following prerequisites in place:
```
Comment (junaid18183) [inline docs/content/docs/how-to-guides/1_deployEnbuildQuickstart.md:22]: why do we need docker ?

File: docs/content/docs/how-to-guides/1_deployEnbuildQuickstart.md [hunk 2]
```diff
@@ -75,7 +77,7 @@ Make sure you update the values input to reference the values you created in Ste
 Execute the command below.
 
 ```bash
-helm upgrade --install --namespace enbuild enbuild vivsoft/enbuild --create-namespace -f target/quick_install.yaml
+helm upgrade --install --namespace enbuild enbuild vivsoft/enbuild --create-namespace -f >path of file</quick_install.yaml
```
Comment (junaid18183) [inline docs/content/docs/how-to-guides/1_deployEnbuildQuickstart.md:80]: change the >path of file</quick_install.yaml to quick_install.yaml

File: docs/content/docs/how-to-guides/2_configuringEnbuild.md [hunk 3]
```diff
@@ -33,6 +33,7 @@ ENBUILD supports GitHub, GitLab VCS as of now.
 These are the Version Control System where ENBUILD creates repositories when you deploy catalog item
 
 ### GITHUB
+Navigate to Admin Settings and select GITHUB under, Set and configure your VCS account. Once selected click the configure button at the top right of the GITHUB box you have selected.
```
Comment (junaid18183) [inline docs/content/docs/how-to-guides/2_configuringEnbuild.md:36]: By default is Github is disabled, To enable the Github first you need to 
- Navigate to Admin Settings and select GITHUB under, Set and configure your VCS account. Once selected click the configure button at the top right of the GITHUB box you have selected

**Review by junaid18183 — CHANGES_REQUESTED** [review]
See changes requested

File: docs/content/docs/how-to-guides/1_deployEnbuildQuickstart.md (hunk 1 above)
Comment (jvaidy3) [reply docs/content/docs/how-to-guides/1_deployEnbuildQuickstart.md:22]: Docker is the containerization platform that manages the k3d clusters. When I tried running the clusters without docker they wouldn't start.

Comment (junaid18183) [reply docs/content/docs/how-to-guides/1_deployEnbuildQuickstart.md:22]: yeah, so we have covered Kubernetes as Prerequisites,  how user creates Kubernetes is upto them. 
k3d is one option.

**Pull request closed without merging**